halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/deck/DeckProfileFactory.java

```diff
@@ -124,7 +124,7 @@ protected void setProfile(Profile profile, DeploymentConfiguration deploymentCon
     if (openstackProvider.getPrimaryAccount() != null) {
       OpenstackAccount openstackAccount = (OpenstackAccount) accountService.getProviderAccount(deploymentConfiguration.getName(), "openstack", openstackProvider.getPrimaryAccount());
       //Regions in openstack are a comma separated list. Use the first as primary.
```

Comment:
This comment is out of date, isn't it? Remove or update. Regions are now a list of strings from command all the way to domain objects.